inc/managers/class-site-manager.php

```
@@ -876,4 +946,418 @@ public function delete_pending_sites(): void {
+		$blog_id    = $site->get_blog_id();
+
+		// Fire pre-deletion hook for extensibility.
+		do_action('wu_before_demo_site_deleted', $site, $membership, $customer);
+
+		// Delete the WordPress blog if it exists.
+		if ($blog_id) {
+			wpmu_delete_blog($blog_id, true);
+		}
+
+		// Delete the WP Ultimo site record.
+		$result = $site->delete();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)